gulpfile.js

```diff
@@ -67,6 +67,27 @@ function getPlatforms() {
     return platforms;
 }
 
+function getRunDebugAppCommand() {
+    switch (os.platform()) {
+    case 'darwin':
+        return 'open ' + path.join(debugDir, pkg.name, 'osx64', pkg.name + '.app');
+
+        break;
+    case 'linux':
+        return path.join(debugDir, pkg.name, 'linux', pkg.name);
```

Comment:
This needs to be `linux64`. Once this is changed it runs just fine under linux. (Also: Good idea!)